bundle/schema/jsonschema.json

```diff
@@ -4365,6 +4357,207 @@
                 }
               ]
             },
+            "serving.AiGatewayConfig": {
```

Comment:
Where does this come from?

Reply:
I think this is an innocent enough user error. If the CLI is first generated by running `genkit generate-sdk cli` and the Go SDK version is updated in `go.mod` then the new fields from the schema are not really generated. That is what I suspect happened here in https://github.com/databricks/cli/pull/1799.

I don't we need any followups here since every time this happens we'll catch up on the next release.

cc: @andrewnester FYI

Reply:
I was able to reproduce this when I tried `genkit` and then `go.mod`. `go.mod` and then `genkit` works correctly.

Reply:
I see, thanks. Can we enforce that somehow?

Reply:
Probably by updating the go SDK version as part of the gen kit. Let me give it a quick try.